src/includes/set-level/unity.mdx

```diff
@@ -0,0 +1,5 @@
+```csharp
+using Sentry;
+
+SentrySdk.CaptureException(new Exception(), s => s.Level = SentryLevel.Warning);
```

Comment:
Global scope vs local scope.

I'd remove this configure scope anyway, since any event sent after this call will take that level now, which is suboptimal.

Level should be set on each exact capture whenever possible. And on Global scope, that should be the only way.